exp-player/addon/components/exp-player.js

```diff
@@ -18,7 +19,19 @@ export default Ember.Component.extend({
         var frameConfigs = parseExperiment(this.get('experiment.structure'));
         this.set('frames', frameConfigs);  // When player loads, convert structure to list of frames
     },
-
+    session: Ember.computed('experiment', function() {
+        var session = this.get('store').createRecord(this.get('experiment.sessionCollectionId'), {
```

Comment:
My understanding of the Ember docs is that by default computed values are cached: http://emberjs.com/api/classes/Ember.ComputedProperty.html#method_volatile.

So the use of a computed here is very deliberate. If the experiment changes, the session should change.